.prettierrc.json

```diff
@@ -1 +0,0 @@
-{}
```

Comment:
I feel like one of my avatars is Prettier cop but since we moved all the substance to `blocks-dev` I'll allow it 😄